core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -250,7 +250,7 @@ private FileSystemContext(AlluxioConfiguration conf, @Nullable BlockWorker block
     mWorkerRefreshPolicy =
         new TimeoutRefresh(conf.getMs(PropertyKey.USER_WORKER_LIST_REFRESH_INTERVAL));
     LOG.debug("Created context with id: {}, with local block worker: {}",
-        mId, mBlockWorker == null);
+        mId, mBlockWorker != null);
```

Comment:
Is this Java syntactic sugar for `mBlockWorker != null ? mBlockWorker : null` or are we intentionally logging a boolean expression result?

Reply:
we are intentionally logging a boolean here as DefaultBlockWorker does not have a `toString()`.